ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -132,6 +138,12 @@ literals:
 - outside_air_relative_humidity_sensor
 - outside_air_temperature_sensor
 - outside_air_wetbulb_temperature_sensor
+- outside_air_fan_run_status:
```

Comment:
are these used elsewhere in the PR? if not, remove them. this goes for all other unused fields and subfields.